.goreleaser.yaml

```diff
@@ -0,0 +1,32 @@
+# This is an example .goreleaser.yml file with some sensible defaults.
+# Make sure to check the documentation at https://goreleaser.com
+before:
+  hooks:
+    # You may remove this if you don't use go modules.
+    - go mod tidy
+    # you may remove this if you don't need go generate
+    - go generate ./...
+builds:
+  - env:
+      - CGO_ENABLED=0
+    goos:
+      - linux
+      - windows
```

Comment:
Do we know whether the program works on Windows?

Reply:
Checking here.

Reply:
Yes, works fine :)

Reply:
i can say for windows amd64, it does.
on a separate note, maybe we should modify the os-matrix test run to include windows and darwins too?

Reply:
Seems wise if we're going to be building for those targets. 🙂  👍🏼